src/foraging_gui/MetaData.ui

```diff
@@ -0,0 +1,1452 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>MetaData</class>
```

Comment:
We can condense the `ui`. However, it might be ok to keep it a little larger because we don't need to keep this metadata window open, and we usually only need to open it once at different time for different rigs.

Reply:
made an issue: https://github.com/AllenNeuralDynamics/dynamic-foraging-task/issues/470